File: node-app/tictactoe/player.js
function player(){
    var obj={};

    obj.name="";
	obj.mark="";
	obj.createPlayer=function(playerNo,name){
		if(playerNo==1){
			obj.mark="X";
		}
		else{
			obj.mark="0";
		}
	    obj.name=name;
    }
    
    return obj;
}
module.exports=player;

File: TicTacToe/app/src/main/java/com/techlabs/paresh/tic_tac_toe/Player.java
package com.techlabs.paresh.tic_tac_toe;

public class Player {
	public String name;
	Mark mark;
	public Player(int player,String name){
		if(player==1){
			mark=Mark.CROSS;
		}
		else{
			mark=Mark.NOUGHT;
		}
		this.name=name;
	}

	public Mark getMark() {
		return mark;
	}

	public String getName() {
		return name;
	}
}
